doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -1295,6 +1295,9 @@ e.g.
 export JAVA_HOME=/<path_to>/<the_cross_built_jdk>
 export PATH="$JAVA_HOME/bin:$PATH"
 ```
+:bulb:
+Please remove `/usr/lib64/ccache` if exists in the PATH it as `/usr/lib64/ccache/gcc` is a symbolic link to `ccache` which is not supported in the compilation of OpenJDK.
```

Comment:
The grammar here needs fixing; it's unclear whether you're advising
* removing `/usr/lib64/ccache` from the `PATH` environment variable, or
* removing the `/usr/lib64/ccache` directory from the file system.

Reply:
Replaced with `remove /usr/lib64/ccache from the PATH environment variable`